Brams branch

Brambuu opened the pull request:
Kek

Brambuu committed:
Create testietest.txt

Added new test file: testietest.txt

Brambuu committed:
Merge branch 'BramsBranch' of https://github.com/larsk2009/EscapeRoom into BramsBranch

Brambuu committed:
Added test code for 2 readers

Added test code for two RFID readers. The "result" of both readers is visualized with leds. The red led will turn on when the result is 0 and the green led will turn on when the result is 1.

Brambuu committed:
Added Visual Studio Project

Brambuu committed:
Update DigitalPuzzle.ino

Added some explanation for each function

TeunKing committed:
drie functies toegevoegd

de compare NFC functie geupdate. PutTagInEEPROM functie toegevoegd.
CalibrateTags toegevoegd.

TeunKing committed:
Merge branch 'BramsBranch' of https://github.com/larsk2009/EscapeRoom into BramsBranch

larsk2009 commented on DigitalPuzzle/DigitalPuzzle.ino:
```diff
@@ -0,0 +1,343 @@
+#include <EEPROM.h>
+#include"rfid1.h"
+
+const int RedPin[2] = {3,4};
```
```suggestion
const int RedPins[2] = {3,4};
```

larsk2009 commented on DigitalPuzzle/DigitalPuzzle.ino:
```diff
@@ -0,0 +1,343 @@
+#include"rfid1.h"
+
+const int RedPin[2] = {3,4};
+const int GreenPin[2] = {2,5};
```
```suggestion
const int GreenPins[2] = {2,5};
```

larsk2009 commented on DigitalPuzzle/DigitalPuzzle.ino:
```diff
@@ -0,0 +1,343 @@
+bool StartValues[6] = {1,1,1,0,0,0};
+bool ReaderResult[8] = {0,0,0,0,0,0,0,0};
+
+uchar serNum[5]; //Array to store your 
```
Gebruik goede variabel namen. Ik denk dat hier serialNumber bedoelt wordt? Het commentaar is ook niet heel lekker hier :)

larsk2009 reviewed (changes requested):
Kijk hier vast naar, ik kijk morgen naar de rest van de commits.

Brambuu committed:
Proccesed Lars' comments and added SerialRead function

Changed some variable names and added a new functions to read from the serial port. This can be used to access the menu (to add/calibrate new tags).

larsk2009 reviewed (approved):
Er zijn nog wat dingetjes die beter kunnen maar voor nu kunnen we het wel even zo laten. Anders kijken we maandag nog even een keer!
